lib/block.js

```diff
@@ -13,7 +13,7 @@ const TitleWrap = styled.header`
   background-color: var(--colors-secondaryBackground);
 `;
 const TitleContent = styled.h2`
-  flex: 1 0 auto;
+  flex: 1 1 auto;
```

Comment:
This allows titles to flex-shrink. I'm curious where this is fixing something visually. Could you add a screenshot of the affected content?

Reply:
1 0 auto:
![Screen Shot 2019-12-17 at 4 46 21 PM](https://user-images.githubusercontent.com/3011231/71036995-ce13db80-20ec-11ea-8b99-2583078791e9.png)

1 1 auto:
![Screen Shot 2019-12-17 at 4 46 31 PM](https://user-images.githubusercontent.com/3011231/71037021-d9ff9d80-20ec-11ea-85a0-c25388d7aaad.png)

Reply:
Got it! Thanks.